src/tools/cgget.c

```diff
@@ -583,6 +599,19 @@ static int fill_empty_controller(struct cgroup * const cg, struct cgroup_control
 		goto out;
 
 	path_len = strlen(path);
+#ifdef WITH_SYSTEMD
+	if (cg->name[0] == '/' && strncmp(path + (path_len - 7), ".scope/", 7) == 0) {
+		snprintf(tmp, FILENAME_MAX, "%s", dirname(path));
+		strncpy(path, tmp, FILENAME_MAX - 1);
+		path[FILENAME_MAX - 1] = '\0';
+	}
+	path_len = strlen(path);
+	if (cg->name[0] == '/' && strncmp(path + (path_len - 6), ".slice/", 6) == 0) {
```

Comment:
The first comparison is made for the `7` bytes, because the path ends with a `/` and that needs to be taken into account  but after trimming the scope cgroup from the path using `dirname()`, which also deletes the `/`, hence the second comparison is done for 6 characters but you are right, that the comparison is misleading and `.slice` should be used instead.